src/main/java/com/exadel/discount/dto/favorite/FavoriteDto.java

```diff
@@ -0,0 +1,13 @@
+package com.exadel.discount.dto.favorite;
+
+import com.exadel.discount.dto.validation.Create;
+import lombok.Data;
+
+import javax.validation.constraints.Null;
+import java.util.UUID;
+
+@Data
+public class FavoriteDto {
+    @Null(groups = Create.class, message = "Favorite id should be null")
```

Comment:
if it should be null always, why its added to DTO?

Reply:
It was needed in previous wrong architecture - Fixed.
Done